packages/tasks/src/local-apps.ts

```diff
@@ -39,36 +45,52 @@ export type LocalApp = {
 			 * And if not (mostly llama.cpp), snippet to copy/paste in your terminal
 			 * Support the placeholder {{GGUF_FILE}} that will be replaced by the gguf file path or the list of available files.
 			 */
-			snippet: (model: ModelData) => string | string[];
+			snippet: (model: ModelData) => Snippet | Snippet[];
```

Comment:
```suggestion
			snippet: (model: ModelData) => string | string[] | Snippet | Snippet[];
```

to be backward compatible (and to compile)

Reply:
Fixed with https://github.com/huggingface/huggingface.js/pull/778/commits/98a263767c4fd37b000ba94bc92dfa5ea82c6399

But I would recommend to move completely to `Snippet` interface, because my idea is to have dedicated "title" to explain what each snippet does.